clang/test/CodeGen/X86/movrs-avx10.2-builtins-error-32.c

```diff
@@ -0,0 +1,98 @@
+// RUN: %clang_cc1 -ffreestanding %s -Wno-implicit-function-declaration -triple=i386-unknown-unknown -target-feature +movrs -target-feature +avx10.2-256 -emit-llvm -verify
```

Comment:
Same thing here, can we remove "-emit-llvm" since we don't care about the output anyway?

Reply:
Done.